plugins/@grouparoo/mysql/__tests__/table-import/import-properties.ts

```diff
@@ -331,7 +331,7 @@ describe("mysql/table/recordProperties", () => {
           sourceMapping,
           aggregationMethod: AggregationMethod.MostRecentValue,
         });
-        expect(values[record.id][properties[0].id][0]).toEqual("Orange");
+        expect(values[record.id][properties[0].id][0]).toEqual(null);
```

Comment:
This test (in all plugins) went from testing a value (Orange) to finding a null.  It's OK because the test below (least recent value) still checks that non-null value (Apple) is found